lib/matyi.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'format_parser'
```

Comment:
@duplamatyi mind removing this file? ;)

Reply:
ah nvm I've got it

Reply:
@bermannoah of course I had to do something silly in the last 100 meters... thanks for noticing!

Reply:
:)